staging/src/k8s.io/client-go/kubernetes/typed/events/v1beta1/fake/fake_event_expansion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Why?

Reply:
it's something written in 2019, that's why